_src/static/sass/base/_variables.scss

```diff
@@ -0,0 +1,156 @@
+/* ============================
+-------------------------------
+ 1. Grid
+ 2. Fonts
+ 3. Colors (Palette)
+ 4. Colors (Application)
+ 5. Timing
+-------------------------------
+============================ */
+
+$baseline:     20px;
+
+
+/* ============================
+1. Grid
+---------------------------- */
+$columns:      12;
+$min-width:    768px;
+$max-width:    1600px;
+
+
+/* ============================
+2. Fonts
+---------------------------- */
+$font-sans:     'Open Sans','Helvetica Neue', Helvetica, Arial, sans-serif;
+$font-serif:    'Bitstream Vera Sans Mono', Consolas, Courier, monospace;
+$font-size:     16px;
```

Comment:
Let's use rems for a unit of measurement here. Bourbon's Pixel to Rems utility (http://bourbon.io/docs/#px-to-rem) is a godsend for this. the syntax would then be ``rem(16)``.